### ISSUE TITLE
nghttp ignores -H if -u is used

### ISSUE THREAD

**Issue by kazuho** [tatsuhiro-t/nghttp2#161]
It seems that when `-u` is used, nghttp does not send the headers specified by `-H`.  Looking at the log emitted by `-v`, the output is:

```
$ nghttp -u -vv -H 'cookie: a=b' http://127.0.0.1:8080/            
[  0.001] Connected
[  0.001] HTTP Upgrade request
GET / HTTP/1.1
Host: 127.0.0.1:8080
Connection: Upgrade, HTTP2-Settings
Upgrade: h2c-14
HTTP2-Settings: AAMAAABkAAQAAP__
Accept: */*
User-Agent: nghttp2/0.7.5-DEV


^C
$ nghttp -vv -H 'cookie: a=b' http://127.0.0.1:8080/ 
[  0.001] Connected
[  0.001] send SETTINGS frame <length=12, flags=0x00, stream_id=0>
          (niv=2)
          [SETTINGS_MAX_CONCURRENT_STREAMS(0x03):100]
          [SETTINGS_INITIAL_WINDOW_SIZE(0x04):65535]
[  0.001] send HEADERS frame <length=41, flags=0x05, stream_id=1>
          ; END_STREAM | END_HEADERS
          (padlen=0)
          ; Open new stream
          :method: GET
          :path: /
          :scheme: http
          :authority: 127.0.0.1:8080
          accept: */*
          accept-encoding: gzip, deflate
          user-agent: nghttp2/0.7.5-DEV
          cookie: a=b
```

**Comment by tatsuhiro-t** [comment]
Yeah, I confirm it as bug.

**Comment by tatsuhiro-t** [comment]
Fix committed via 512aa89

**Comment by kazuho** [comment]
Thanks a lot!
